src/query/storages/fuse/src/constants.rs

```diff
@@ -27,6 +27,7 @@ pub const FUSE_TBL_SNAPSHOT_STATISTICS_PREFIX: &str = "_ts";
 pub const FUSE_TBL_LAST_SNAPSHOT_HINT: &str = "last_snapshot_location_hint";
 pub const FUSE_TBL_VIRTUAL_BLOCK_PREFIX: &str = "_vb";
 pub const FUSE_TBL_AGG_INDEX_PREFIX: &str = "_i_a";
+pub const FUSE_TBL_INVERTED_INDEX_PREFIX: &str = "_i_f";
```

Comment:
What's the `f` mean in `_i_f`?

Reply:
Does it mean: `f` -- fulltext?

Reply:
`f` means fulltext, chang to `_i_i` is more suitable.